README.md

```diff
@@ -318,6 +318,31 @@ To detect the currently used localization in your tests, use the following code:
 You can access the language using the `deviceLanguage` variable.
 ```
 
+## Prefilling
+
+- You can prefill data within your app using [HSTestingBackchannel](https://github.com/ConfusedVorlon/HSTestingBackchannel)
+
+## Passing a Preprocessor Macro
+
+Snapshot automatically sets the environment variable FASTLANE_SNAPSHOT=1
```

Comment:
I'm confused, is this automatically now, or does the user have to update the project?